review-me/review-me.js

```diff
@@ -0,0 +1,31 @@
+// hello.js
+
+var message = "Hello, world!"
```

Comment:
3  :  MESSAGE  is  not  defined  Reference Error  at  runtime  due  to  case  sensitivity ,  should  be  message